collector/benchlib/src/measure/perf_counter/unix.rs

```diff
@@ -77,19 +77,19 @@ Try lowering it with `sudo bash -c 'echo -1 > /proc/sys/kernel/perf_event_parano
 }
 
 fn prepare_counters(group: &mut Group) -> anyhow::Result<Counters> {
-    let mut add_event = |event: Hardware| {
-        Builder::new()
-            .group(group)
-            .kind(event)
-            .build()
-            .map_err(|error| anyhow::anyhow!("Could not add counter {:?}: {:?}", event, error))
+    let mut add_event = |event: Hardware| match Builder::new().group(group).kind(event).build() {
+        Ok(counter) => Some(counter),
+        Err(error) => {
+            log::warn!("Could not add counter {:?}: {:?}", event, error);
```

Comment:
Expand this message to give a better idea of what might have gone wrong (i.e. hardware doesn't support this counter).